packages/components/src/test/properties/inputs/NumericPropertyInput.test.tsx

```diff
@@ -267,6 +274,18 @@ describe("<NumericInput />", () => {
     expect(spy).to.be.be.calledOnce;
   });
 
+  it("should focus on input if setFocus is true", async () => {
+    const { getByRole } = render(<NumericInput onChange={() => {}} value="1" setFocus={true} />);
+    let inputFocused = false;
+
+    const input = await waitFor(() => getByRole("textbox"));
+    input.addEventListener("focusout", (event) => {
```

Comment:
Would it be possible to use `document.activeElement` to check if input is focused instead of listening for this event?

Reply:
While for numeric property input it does work, for the quantity input it returns it with body, which should not be the case.

Reply:
If `activeElement` is set to `body` does that mean the input is not focused?

Reply:
Adjusted.